Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM httpd:2.4.41-alpine
 ENV AWSTATS_VERSION 7.7-r0
 ENV MOD_PERL_VERSION 2.0.11
 
-RUN apk add --no-cache awstats=${AWSTATS_VERSION} gettext \
+RUN apk add --no-cache awstats=${AWSTATS_VERSION} gettext tzdata \
```

Comment:
This is a great idea. I always was too lazy to take care about setting the correct timezone. 
Unfortunately, just installing the `tzdata` package is not enough to make it work. There need to be a global environment variable `TZ` set, with the value of your actual timezone. 
Could you please add another line in the readme mentioning to the pass your timezone to the `docker run` command - eg: `--env TZ=Europe/Berlin`?

Reply:
Hey, just added a couple of lines on the docs to say that. I chose Antarctica because it's the only continent with all the timezones :)

Reply:
Awesome - thank you :)